docs/setting-up-your-environment.md

```diff
@@ -97,13 +114,33 @@ To preview the website in your local browser, make sure you're in the
 `btcinformation.org` directory and run the following command:
 
     make preview
+    
+###### Mac differences:
```

Comment:
I think it's unclear where this MacOS section ends, e.g.:

![2018-08-07-02_52_05_303971014](https://user-images.githubusercontent.com/61096/43759362-fb2ffd52-99ec-11e8-986c-4ad2b3d2e59e.png)

I suggest combining this information down a bit to under the original "Server running..." line, possibly combining it with the final Mac Differences section in your diff.

Reply:
Ok, thanks for the feedback. Let me give it a second try to consolidate it.

Reply:
Actually, as per @kek-coin 's comments below I'll create a new issue for this section.